docker/autoware-universe/Dockerfile

```diff
@@ -53,14 +53,70 @@ RUN chmod 644 /etc/glvnd/egl_vendor.d/10_nvidia.json
 RUN echo "source /opt/ros/${ROS_DISTRO}/setup.bash" > /etc/bash.bashrc
 CMD ["/bin/bash"]
 
-FROM devel as prebuilt
+FROM devel as builder
```

Comment:
If you don't use the `devel` image for the `prebuilt` image (or is it better to rename it to `runtime` to be consistent with CUDA's images), it might be better to prepare another Dockerfile.